src/appdistribution/client.ts

```diff
@@ -42,10 +53,12 @@ export class AppDistributionClient {
 
   constructor(private readonly appId: string) {}
 
-  async getApp(): Promise<AppDistributionApp> {
-    utils.logBullet("getting app details...");
-
-    const apiResponse = await api.request("GET", `/v1alpha/apps/${this.appId}`, {
+  async getApp(
+    distributionFileType: DistributionFileType = DistributionFileType.APK
+  ): Promise<AppDistributionApp> {
+    utils.logBullet(`getting app details (Distribution type: ${distributionFileType})...`);
```

Comment:
Do we need to specify the Distribution type here? Will the developer know what it means in the context of getting the app details?

Reply:
That's a good question. I could see this as a useful debugging tool for us, but it's unclear if it would be useful for the customer

Reply:
we can get rid of this. I added it mainly for debugging. Not sure if we want to expose this to the customer (yet)

Reply:
Actually, thinking about this more, I think this is useful. Going to leave it.